src/components/endpoint/Endpoint/Endpoint.module.scss

```diff
@@ -43,69 +43,79 @@
+                        background-size: 10px;
                         padding: 0 4px;
                         transform: translateY(-20px) scale(0.75);
                     }
+                    &:focus {
+                        outline: none;
+                        & ~ label {
+                            color: var(--brand-secondary);
+                            background-color: var(--white);
+                            padding: 0 4px;
+                            transform: translateY(-20px) scale(0.75);
+                        }
+                    }
+                }
+                .inlineLabel {
+                    position: relative;
+                    padding: 4px 2px;
+                    display: inline-block;
+                    font-size: 16px;
+                    margin-right: 5px;
+                    line-height: 20px;
+                    color: var(--black-01);
+                    padding: 4px 2px;
+                    background-color: var(--white);
+                    pointer-events: none;
+                    white-space: nowrap;
+                    transform: translate(6px, 10px);
+                }
+                .textInput {
+                    background: none;
+                    border: 1px solid #ccc;
+                    color: var(--black-01);
+                    padding: 1rem 1rem 1rem 0.8rem;
+                    width: 100%;
+                }
+                .errorMessage {
+                    color: var(--red-01);
+                    font-size: 14px;
+                    font-weight: 400;
+                    padding-left: 10px;
                 }
-            }
-
-            .inlineLabel {
-                right: 11rem;
-                top: 10px;
-                position: relative;
-                padding: 4px 2px;
-                display: flex;
-                font-size: 16px;
-                margin-right: 5px;
-                line-height: 20px;
-                color: var(--black-01);
-                padding: 4px 2px;
-                background-color: var(--white);
-                pointer-events: none;
-                white-space: nowrap;
-            }
-            .textInput {
-                background: none;
-                border: 1px solid #ccc;
-                color: var(--black-01);
-                font-size: var();
-                padding: 1rem 1rem 1rem 0.8rem;
-                width: 25%;
-            }
-            .errorMessage {
-                color: var(--red-01);
-                font-size: 14px;
-                font-weight: 400;
-                padding-left: 10px;
             }
         }
         .url {
             display: flex;
             flex-direction: row;
             font-size: 20px;
             padding: 10px;
+            @media (max-width: 785px) {
+                flex-direction: column;
+            }
+
+            @media (max-width: 650px) {
+                text-align: center;
+            }
```

Comment:
How come these have media queries for these specifically? Is it not possible to put them into a data state?